catalog/core/catalog-core-impl/filter-proxy/src/main/java/ddf/catalog/filter/proxy/adapter/GeotoolsFilterAdapterImpl.java

```diff
@@ -472,6 +479,20 @@ public Object visit(PropertyIsGreaterThanOrEqualTo filter, Object delegate) {
             return lessThanOrEqual.accept(this, delegate);
         }
 
+        //Special case for handling temporal queries
+        if (literal instanceof String && RELATIVE_TEMPORAL_MATCHER.matcher((String) literal)
+                .matches()) {
+            DateTime currentDateTime = new DateTime();
+
+            String duration = (((String) literal).split("[\\(\\)]"))[1];
```

Comment:
Short story is that I'm more familiar with String methods than regex and knew this would work 😄 
If there's a performance gain the other way I'll be happy to change it